<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjetGeometrieClass</class>
 <widget class="QWidget" name="ProjetGeometrieClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjetGeometrie</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="BoutonEquerre">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Resources/equerre.gif</normaloff>Resources/equerre.gif</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="BoutonCrayon">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Resources/crayon.gif</normaloff>Resources/crayon.gif</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeatDelay">
         <number>100</number>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="BoutonPleinEcran">
        <property name="icon">
         <iconset>
          <normaloff>Resources/SortieFullscreen.gif</normaloff>Resources/SortieFullscreen.gif</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="3">
       <widget class="QCheckBox" name="CheckBoxGrille">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Grille</string>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="BoutonPoint">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Resources/Point.gif</normaloff>Resources/Point.gif</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="BoutonNoir">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background : rgb(0, 0, 0)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QPushButton" name="BoutonTexte">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Resources/Texte.gif</normaloff>Resources/Texte.gif</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="10" column="0" rowspan="3">
       <widget class="QPushButton" name="BoutonEpaisseur">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Resources/Trait.gif</normaloff>Resources/Trait.gif</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="BoutonRegle">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Resources/regle.gif</normaloff>Resources/regle.gif</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="BoutonCompas">
        <property name="icon">
         <iconset>
          <normaloff>Resources/compas.gif</normaloff>Resources/compas.gif</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="DessinLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="ScrollAreaOptionsOutils">
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>227</width>
        <height>872</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QDockWidget" name="DockWidgetCrayon">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>162</width>
           <height>156</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>524287</width>
           <height>160</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 203, 96);</string>
         </property>
         <property name="floating">
          <bool>false</bool>
         </property>
         <property name="features">
          <set>QDockWidget::AllDockWidgetFeatures</set>
         </property>
         <property name="allowedAreas">
          <set>Qt::AllDockWidgetAreas</set>
         </property>
         <property name="windowTitle">
          <string>Crayon</string>
         </property>
         <widget class="QWidget" name="dockWidgetContents_3">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="GroupBoxCrayonPosition">
             <property name="title">
              <string>Position</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="LabelPositionX">
                <property name="text">
                 <string>x</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SpinBoxCrayonPositionX">
                <property name="maximum">
                 <number>1000</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="LabelPositionY">
                <property name="text">
                 <string>y</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SpinBoxCrayonPositionY">
                <property name="maximum">
                 <number>1000</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="CheckBoxCrayonTransparence">
             <property name="text">
              <string>transparence</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="WidgetCrayonOrientation" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="LabelCrayonOrientation">
                <property name="text">
                 <string>Orientation</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="SpinBoxCrayonOrientation">
                <property name="minimum">
                 <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>359.990000000000009</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QDockWidget" name="DockWidgetRegle">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>182</width>
           <height>217</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>524287</width>
           <height>220</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(169,234,254);</string>
         </property>
         <property name="floating">
          <bool>false</bool>
         </property>
         <property name="features">
          <set>QDockWidget::AllDockWidgetFeatures</set>
         </property>
         <property name="windowTitle">
          <string>Règle</string>
         </property>
         <widget class="QWidget" name="dockWidgetContents_4">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QGroupBox" name="GroupBoxReglePosition">
             <property name="title">
              <string>Position</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="LabelReglePositionX">
                <property name="text">
                 <string>x</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SpinBoxReglePositionX">
                <property name="maximum">
                 <number>1000</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="LabelReglePotitionY">
                <property name="text">
                 <string>y</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SpinBoxReglePositionY">
                <property name="maximum">
                 <number>1000</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="CheckBoxRegleTransparence">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>transparence</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="LabelRegleOrientation">
                <property name="text">
                 <string>Orientation</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="SpinBoxRegleOrientation">
                <property name="minimum">
                 <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>359.990000000000009</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>55</height>
              </size>
             </property>
             <property name="title">
              <string>Tracer</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QDoubleSpinBox" name="SpinBoxRegleTracerPoint1"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="SpinBoxRegleTracerPoint2"/>
              </item>
              <item>
               <widget class="QPushButton" name="BoutonRegleTracerOK">
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>OK</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QDockWidget" name="DockWidgetEquerre">
         <property name="minimumSize">
          <size>
           <width>182</width>
           <height>220</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>524287</width>
           <height>220</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(135,233,144);</string>
         </property>
         <property name="features">
          <set>QDockWidget::AllDockWidgetFeatures</set>
         </property>
         <property name="windowTitle">
          <string>Equerre</string>
         </property>
         <widget class="QWidget" name="dockWidgetContents_5">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="GroupBoxEquerrePosition">
             <property name="title">
              <string>Position</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="LabelEquerrePositionX">
                <property name="text">
                 <string>x</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SpinBoxEquerrePositionX">
                <property name="maximum">
                 <number>1000</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="LabelEquerrePositionY">
                <property name="text">
                 <string>y</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SpinBoxEquerrePositionY">
                <property name="maximum">
                 <number>1000</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="CheckBoxEquerreTransparence">
             <property name="text">
              <string>transparence</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="WidgetEquerreOrientation" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="LabelEquerreOrientation">
                <property name="text">
                 <string>Orientation</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="SpinBoxEquerreOrientation">
                <property name="minimum">
                 <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>359.990000000000009</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="GroupBoxEquerreTracer">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>60</height>
              </size>
             </property>
             <property name="title">
              <string>Tracer</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QDoubleSpinBox" name="SpinBoxEquerreTracerPoint1"/>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="SpinBoxEquerreTracerPoint2"/>
              </item>
              <item>
               <widget class="QPushButton" name="BoutonEquerreTracerOK">
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>OK</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QDockWidget" name="DockWidgetCompas">
         <property name="minimumSize">
          <size>
           <width>209</width>
           <height>231</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>524287</width>
           <height>235</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(240,227,107);</string>
         </property>
         <property name="features">
          <set>QDockWidget::AllDockWidgetFeatures</set>
         </property>
         <property name="windowTitle">
          <string>Compas</string>
         </property>
         <widget class="QWidget" name="dockWidgetContents_2">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="GroupBoxPositionCompas">
             <property name="title">
              <string>Position</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="LabelCompasPositionX">
                <property name="text">
                 <string>x</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SpinBoxCompasPositionX">
                <property name="maximum">
                 <number>1000</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="LabelCompasPositionY">
                <property name="text">
                 <string>y</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SpinBoxCompasPositionY">
                <property name="maximum">
                 <number>1000</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="WidgetCompasOption" native="true">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="1" column="0">
               <widget class="QLabel" name="LabelEcartementCompas">
                <property name="text">
                 <string>Ecartement</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="SpinBoxCompasEcartement">
                <property name="maximum">
                 <number>350</number>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="LabelCompasOrientation">
                <property name="text">
                 <string>Orientation</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="SpinBoxCompasOrientation">
                <property name="minimum">
                 <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>359.990000000000009</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="GroupBoxCompasTracer">
             <property name="title">
              <string>Tracer</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="labelCompasAngleArriver">
                <property name="text">
                 <string>Angle</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="SpinBoxCompasTracerArc">
                <property name="maximum">
                 <double>359.990000000000009</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="BoutonCompasTracerOK">
                <property name="text">
                 <string>OK</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
